### PR TITLE
[cli] create_or_update_cluster fix

### LINKED ISSUE #10082: [autoscaler] ray submit --start is broken
### What is the problem?

`ray submit --start ` is broken on latest master (presumably this is due to recent logging changes)

### Reproduction (REQUIRED)
Please provide a script that can be run to reproduce the issue. The script should have **no external library dependencies** (i.e., use fake or mock data / environments):

```
(base) Alexs-MacBook-Pro-2:ray alex$ ray submit --start multi.yaml test.py 
Traceback (most recent call last):
  File "/Users/alex/miniconda3/bin/ray", line 11, in <module>
    load_entry_point('ray', 'console_scripts', 'ray')()
  File "/Users/alex/anyscale/ray/python/ray/scripts/scripts.py", line 1587, in main
    return cli()
  File "/Users/alex/miniconda3/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/alex/miniconda3/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/alex/miniconda3/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alex/miniconda3/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/alex/miniconda3/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/alex/anyscale/ray/python/ray/scripts/scripts.py", line 1237, in submit
    True, cluster_name, False)
TypeError: create_or_update_cluster() missing 2 required positional arguments: 'dump_command_output' and 'use_login_shells'
```

If we cannot run your script, we cannot fix your issue.

- [ ] I have verified my script runs in a clean environment and reproduces the issue.
- [ ] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/latest/installation.html).

### PR DESCRIPTION
**Pull request by richardliaw**
## Why are these changes needed?

submit --start was broken.

## Related issue number
closes https://github.com/ray-project/ray/issues/10082

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Review by wuisawesome — APPROVED** [review]

File: python/ray/autoscaler/commands.py [hunk 1]
```diff
@@ -94,12 +94,16 @@ def request_resources(num_cpus=None, bundles=None):
...
 
 
-def create_or_update_cluster(
-        config_file: str, override_min_workers: Optional[int],
-        override_max_workers: Optional[int], no_restart: bool,
-        restart_only: bool, yes: bool, override_cluster_name: Optional[str],
-        no_config_cache: bool, dump_command_output: bool,
-        use_login_shells: bool) -> None:
+def create_or_update_cluster(config_file: str,
+                             override_min_workers: Optional[int],
```
Comment (ijrsvt) [inline python/ray/autoscaler/commands.py:98]: Is it too much to change this to `create_or_update_cluster(*, config_file: str....)`

Comment (wuisawesome) [reply python/ray/autoscaler/commands.py:98]: we should probably keep the hotfix minimal, but should have a discussion about these function signatures

Comment (richardliaw) [reply python/ray/autoscaler/commands.py:98]: I think that'd break the product even more :)

Comment (richardliaw) [reply python/ray/autoscaler/commands.py:98]: but yeah we should raise a warning saying that this will be done

**Pull request merged**